mrbgems/mruby-io/mrbgem.rake

```diff
@@ -9,4 +9,11 @@ MRuby::Gem::Specification.new('mruby-io') do |spec|
     spec.linker.libraries << "ws2_32"
   end
   spec.add_test_dependency 'mruby-time', core: 'mruby-time'
+
+  spec.terms << {
+    "README.md" => proc {
+      pattern = /Copyright \(c\) 2013 Internet Initiative Japan Inc\..*?DEALINGS IN THE SOFTWARE\.\n/m
```

Comment:
@dearblue What happens if we change the copyright year?

Reply:
I think that part should not be touched by mruby developpers, so I don't think there will be any changes.
If it still changes, we will get an error by `File.extract` during the mruby build. :yum:

I think I should fix it when I really need it.